### ISSUE TITLE
FIXME: app crash with right click "Opening url in browser" on 1.0.5.2

### ISSUE THREAD

**Issue by kn007** [RimSort/RimSort#99]
The following log was found from in user_temp\rimsort.log after the crash
```
[INFO][2023-06-09 20:46:13][37504][model.mod_list][eventFilter][123] : USER ACTION: Open right-click mod_list_item contextMenu
[DEBUG][2023-06-09 20:46:13][37504][model.mod_list][eventFilter][178] : Tracking PublishedFileID for ISteamUGC/UnsubscribeItem: 1713367505
[INFO][2023-06-09 20:46:14][37504][model.mod_list][eventFilter][279] : Opening url in browser: https://steamcommunity.com/sharedfiles/filedetails/?id=1713367505
```

the 1.0.5.1 working fine.

**Comment by kn007** [comment]
log: [RimSort_open_url_in_browser.log](https://kn007.net/filecup/uploads/dc1747d888ddcb7af1e8d97c1497d271.log)

step:
1.open rimsort
2.pick a mod to right click
3.click "Opening url in browser"

app exit.

**Comment by twstagg** [comment]
> log: [RimSort_open_url_in_browser.log](https://kn007.net/filecup/uploads/dc1747d888ddcb7af1e8d97c1497d271.log)
> 
> step: 1.open rimsort 2.pick a mod to right click 3.click "Opening url in browser"
> 
> app exit.
> 
> working fine with version below then 1.0.5.2

Thanks. I am able to reproduce as well. I will look into this

**Comment by twstagg** [comment]
Fixed in cd62526df135a10e2d9463f772dfb570aa810ce4
